<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>659</width>
    <height>596</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="imgLabel1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>301</width>
     <height>451</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="imgLabel2">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>130</y>
     <width>301</width>
     <height>451</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="c1">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>80</y>
     <width>70</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>head</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="c2">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>80</y>
     <width>70</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>eyes</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>20</y>
     <width>571</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="button1">
      <property name="text">
       <string>Load Image</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="button2">
      <property name="text">
       <string>Save Image</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="button3">
      <property name="text">
       <string>Get</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QSlider" name="hSlider">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>90</y>
     <width>371</width>
     <height>22</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
